google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -187,6 +193,26 @@ public static BlobSourceOption decryptionKey(String key) {
   }
 
   /**
+   * Downloads this blob to the given file path.
+   *
+   * @param path destination
+   * @throws IOException upon failure
+   */
+  public void downloadTo(Path path) throws IOException {
+    FileOutputStream outputStream = new FileOutputStream(path.toFile());
+    try (ReadChannel reader = reader()) {
+      WritableByteChannel channel = Channels.newChannel(outputStream);
+      ByteBuffer bytes = ByteBuffer.allocate(64 * 1024);
```

Comment:
I'm afraid that 64k is too small for such operation. The actual reader chooses the maximum of the capacity of the supplied ByteBuffer and [DEFAULT_CHUNK_SIZE](https://github.com/vam-google/google-cloud-java/blob/master/google-cloud-storage/src/main/java/com/google/cloud/storage/BlobReadChannel.java#L41), which is 2Mb. I believe it would make sence to make the byte buffer to match the default chunk size (I see that 64 * 1024 was chosen from the documentation of the `Blob#read()`, but the doc is for external usage, for internal usage we can stick to the actually used buffer sizes).